sdk/healthinsights/azure-health-insights-radiologyinsights/README.md

```diff
@@ -67,9 +72,26 @@ Build an **asynchronous** client:
 String endpoint = Configuration.getGlobalConfiguration().get("AZURE_HEALTH_INSIGHTS_ENDPOINT");
 String apiKey = Configuration.getGlobalConfiguration().get("AZURE_HEALTH_INSIGHTS_API_KEY");
 
-RadiologyInsightsAsyncClient radiologyInsightsAsyncClient = new RadiologyInsightsClientBuilder()
-        .endpoint(endpoint).serviceVersion(RadiologyInsightsServiceVersion.getLatest())
-        .credential(new AzureKeyCredential(apiKey)).buildAsyncClient();
+RadiologyInsightsClientBuilder clientBuilder = new RadiologyInsightsClientBuilder()
+        .endpoint(endpoint).serviceVersion(RadiologyInsightsServiceVersion.getLatest());
+if (apiKey != null && !apiKey.equals("")) {
+    clientBuilder = clientBuilder.credential(new AzureKeyCredential(apiKey));
+}
+RadiologyInsightsAsyncClient radiologyInsightsAsyncClient = clientBuilder.buildAsyncClient();
+```
+
+#### Authenticate and create a RadiologyInsightsClient with DefaultAzureCredential
+
+DefaultAzureCredential provides an alternative way to authenticate with the service. Documentation about this can be found [here][azure_credential]
+
+```java com.azure.health.insights.radiologyinsights.defaultazurecredential
+String endpoint = Configuration.getGlobalConfiguration().get("AZURE_HEALTH_INSIGHTS_ENDPOINT");
+
+RadiologyInsightsClientBuilder clientBuilder = new RadiologyInsightsClientBuilder()
+        .endpoint(endpoint).serviceVersion(RadiologyInsightsServiceVersion.getLatest());
+DefaultAzureCredential c = new DefaultAzureCredentialBuilder().build();
```

Comment:
Use full var name, e.g. `credential`.

Also, I would suggest put the AAD (MS Entra ID) section before API-key section. Azure would recommend OAuth over api-key.

Reply:
Changed as suggested